spec/models/chargeback_container_image_new_chargeback_spec.rb

```diff
@@ -0,0 +1,3 @@
+Settings[:new_chargeback] = '1'
```

Comment:
We really should not be messing with globals like this...this is guaranteed to be a sproadic test failure depending on which runs first...  cc @jrafanie